[IMP] pos/multi-employee: update Multi-employee management page 19.0 (#18799)
odoo/documentation: content/applications/sales/point_of_sale/extra/employee_login.rst

Reviewer: does this involve presets as well? and by this i mean: are presets impacted by any of these access rights settings?

Author: All of them can manage presets, only thos with **Advanced rights** can create them, because they can access the backend

Multi-employee management
=========================

The :guilabel:`Log in with Employees` setting allows multiple users to log into the :ref:`POS

> register <pos/use/open-register>`. Activating this setting enables the following actions:

Reviewer: - I wouldn't mention/name the setting directly in the intro like this and instead write something a bit more general saying pos can be configured to allow blabla or something similar

- And is the part *allows multiple users to log into the POS* phrasing completely accurate? I mean, even without the log in with employees settings, the different users are able to connect to the POS, correct? doesn't the setting just allow to actually "track" who is connecting to the POS and configure "access rights"? I'm not sure my queston is 100% clear, let me know if it isn't!

- Selecting the specific users who can :ref:`log into the POS <pos/employee_login/use>`.
- :ref:`Assigning minimal, basic, or advanced permissions <pos/employee_login/configuration>` to
  these users.
- :ref:`Tracking the employees involved in each order for enhanced analytics <pos/analytics>`.

.. _pos/employee_login/configuration:

Configuration
=============

> To allow multiple employees to connect to the same point of sale, go to :menuselection:`Point of

Reviewer: same question as the second list item in [this comment](https://github.com/odoo/documentation/pull/18799/changes#r3542647676)

Sale --> Configuration --> Settings`, scroll down to the :guilabel:`PoS Interface` section, enable
:guilabel:`Log in with Employees`, and click :guilabel:`Save`.

Scroll back to the :guilabel:`PoS Interface` section and complete the :guilabel:`Log in with

> Employees` setting by following the next steps:

Reviewer: *complete* a setting sounds a bit odd to me:

```suggestion
Scroll back to the :guilabel:`PoS Interface` section and the :guilabel:`Log in with
Employees` setting, then complete the following fields:
```

- Add the employees with **extended POS functionality** access in the :guilabel:`Advanced rights`
  field.
- Add the employees with **basic POS functionality** access in the :guilabel:`Basic rights` field.
- Add the employees with **minimal POS functionality** access in the :guilabel:`Minimal rights`

>   field.

Reviewer: Then I would rephrase each list item this to have:

- :guilabel:`field label`: description

for example:

```suggestion
- :guilabel:`Advanced rights`: Add the employees with **extended POS functionality** access.
```

.. image:: employee_login/activate-setting.png
   :alt: setting to enable multiple cashiers in POS

> 

Reviewer: i'd remove this screenshot

.. note::
   - Leaving the :guilabel:`Minimal rights` and :guilabel:`Basic rights` fields empty allows all
     employees to log in.
   - Leaving the :guilabel:`Advanced rights` field empty grants extended rights to Odoo users only.

> 

Reviewer: What about this?

<img width="474" height="146" alt="Image" src="https://github.com/user-attachments/assets/0f2e8651-79e8-41dc-b7d1-fd4514d29a87" />

      - Lock and unlock an open POS register.
      - Reload data.

>       - Print reports.

Reviewer: what reports? Do we mean reports in the studio sense (i.e., documents?), so for example, invoices and so on? or do we mean an actual report somewhere (if so, where and which one(s))?

Author: I'll add a ref to the Reporting doc

      - Enable :doc:`../hardware_network/pos_lna`.
      - Access the :ref:`orders overview <pos/use/orders>`, search, and filter orders.
      - Print or reprint :doc:`invoices <../use/pos_invoices>` and :doc:`receipts <../use/receipts>`
        in the orders overview.

      **Sales transactions:**

      - :ref:`Process standard sales transactions <pos/use/sell>`.

>       - :ref:`Select customers <pos/use/customers>`.

Reviewer: I'd rephrase this to be a bit more precise ("select customers" is pretty general), maybe even use the related section heading?

```suggestion
      - :ref:`Register and assign customers <pos/use/customers>`.
```

Author: They can't register customers with minimal rights, but I'll go for **assign**, that's what I meant with **select**, thanks 😄

----------

Once the :guilabel:`Log in with Employees` setting is enabled, employees must log in to :ref:`open
the POS register <pos/use/open-register>` and access the POS interface. They can :ref:`scan their
employee badge <pos/employee_login/badge>`, click the :icon:`fa-users` icon (:guilabel:`users`) to
select their name from the list of authorized users, or enter :ref:`their PIN code
<pos/employee_login/pin>` in the :guilabel:`Enter your PIN` field.

.. image:: employee_login/log-in.png
   :alt: Login window to open the register when the multiple cashiers feature is active

To switch between users from the :ref:`POS register <pos/use/open-register>`, click the currently
logged-in employee's name at the top right of the POS interface and select the user to switch to.

.. tip::
   In the absence of a scanner, click the :icon:`fa-barcode` icon (:guilabel:`barcode`) to scan
   barcodes using your device's webcam.

.. _pos/employee_login/badge:

Badge login
-----------

Employees can log in using their badge. To configure badge-based login, assign a unique badge ID to
the employee's profile in the :guilabel:`Employees` app:

#. Navigate to the :guilabel:`Employees` module.

> #. Click the relevant employee.

Reviewer: ```suggestion
#. Open the Employees app and select the relevant employee.
```

#. Go to the :guilabel:`Settings` tab.
#. The :guilabel:`Attendance/Point of Sale/Manufacturing` category offers two options:

   - Manually enter any badge ID in the :guilabel:`Badge ID` field.
   - Click :guilabel:`Generate` to automatically generate a unique badge ID.

> #. Click :guilabel:`Print Badge` to generate a barcode representation of the assigned badge ID.

Reviewer: Since this is actually configuration, not use, I'd move this sub-section to the Configuration section, and just keep the "use part" (below, lines 144-150) here I think

To switch between users from the :ref:`POS register <pos/use/open-register>`, using a badge, first
lock the register. To do so, click the :icon:`fa-lg fa-lock` icon (:guilabel:`lock`) to return to

> the login screen. Then, the new employee can scan their badge to log in.

Reviewer: ```suggestion
To switch between users from the :ref:`POS register <pos/use/open-register>` using a badge first, lock the register by clicking the :icon:`fa-lg fa-lock` icon (:guilabel:`lock`) in the upper-right corner. The new employee can then scan their badge to log in.
```

.. note::

>    If a PIN code is associated with a user, they will be asked to enter the it even after scanning

Reviewer: ```suggestion
   If a PIN code is associated with a user, they will be asked to enter it even after scanning
```

   the badge.

.. _pos/employee_login/pin:

> PIN code setup

Reviewer: Since this is actually configuration, not use, I'd move this sub-section to the Configuration section

and see my general comment about merging the configuration sections for PIN code and badge login

--------------

For enhanced security, employees may be required to enter the PIN code each time they log in to the
POS register. To set up a PIN code for an employee:

#. Navigate to the **Employees** app.

> #. Click the relevant employee.

Reviewer: ```suggestion
#. Open the Employees app and select the relevant employee.
```
